recipes/gtk/all/conanfile.py

```diff
@@ -74,8 +87,6 @@ def configure(self):
             if self.options.with_wayland or self.options.with_x11:
                 if not self.options.with_pango:
                     raise ConanInvalidConfiguration("with_pango option is mandatory when with_wayland or with_x11 is used")
-        if self.settings.os == "Windows":
-            raise ConanInvalidConfiguration("GTK recipe is not yet compatible with Windows. Contributions are welcome.")
 
     def build_requirements(self):
         self.build_requires("meson/0.59.1")
```

Comment:
```suggestion
        self.build_requires("meson/0.60.2")
        self.build_requires("libxml2/2.9.12") # for xmllint
```
This should make the windows build go one step further